lib/src/task_process.dart

```diff
@@ -57,43 +57,37 @@ class TaskProcess {
   Stream<String> get stderr => _stderr.stream;
   Stream<String> get stdout => _stdout.stream;
 
-  Future<List<int>> getChildPids(List<int> pids) async {
+  bool kill([ProcessSignal signal = ProcessSignal.SIGTERM]) =>
+      _process.kill(signal);
+
+  Future<bool> killAllChildren(
+      [ProcessSignal signal = ProcessSignal.SIGTERM]) async {
```

Comment:
Previously this method was expecting a process id as an arg, but since this is a method on a `TaskProcess` instance, it makes more sense for it to kill all children of that process. I also added the `signal` param to be consistent with the `kill()` method